h2o-py/test-requirements.txt

```diff
@@ -32,17 +34,19 @@ sphinx_rtd_theme==0.2.4
 sphinxcontrib-osexample>=0.1.1
 shap==0.29.1
 boto3==1.18.65
-lifelines>=0.19.5
+lifelines<0.27; python_version < '3.9'
+lifelines>=0.27; python_version >= '3.9'
 xlsxwriter==1.3.7
-xgboost==1.7.6
+xgboost==1.7.6; python_version >= '3.8'
 causalml==0.12.3; python_version <= '3.7'
 causalml==0.14.1; python_version > '3.7' and python_version < '3.11'
 colorama
+datatable==1.0.0; python_version <= '3.9'
+polars==0.19.17; python_version > '3.9'
+pyarrow==14.0.1; python_version > '3.9'
 # not directly required, pinned by Snyk to avoid a vulnerability
-pygments==2.7.4; python_version <= '3.7' 
-pygments==2.15.1; python_version > '3.7'
-sphinx==3.0.4; python_version <= '3.8'
+pygments==2.7.4; python_version < '3.5'
```

Comment:
this line can be deleted, Python 3.5 is not currently supported

Reply:
Thanks, I wasn't sure if we don't use python 2.7 for building the documentation (pygments does syntax highlighting) so I kept it here. I think we don't but could you confirm that?

Reply:
The release pipeline uses python 3.6: https://github.com/h2oai/h2o-3/blob/master/scripts/jenkins/jenkinsfiles/Jenkinsfile-Release#L20